### ISSUE TITLE
Subscribed, want to cancel

### ISSUE THREAD

**Issue by AxDSan** [shobrook/adrenaline#54]
I subscribed to the online useadrenaline service, but when I go to Manage Account, to cancel the subscription it won't let me... contact me ASAP, as I don't find a practical usage for this.

**Comment by dillfrescott** [comment]
Good thing i didnt subscribe. Just using the demo, this thing doesnt help with anything

**Comment by AxDSan** [comment]
> Good thing i didnt subscribe. Just using the demo, this thing doesnt help with anything

Yup how convenient that this is one specific feature is not working...

**Comment by dillfrescott** [comment]
Try to refute the charge perhaps

**Comment by AxDSan** [comment]
Ik, that’s what I’ve been thinking about but honestly just trying to evade
having to call my bank and go through the entire ordeal, also when I open a
dispute they require me to close/suspend my card and I don’t want to go
through that… really sucks

**Comment by dillfrescott** [comment]
Next time use a service like PayPal. They make it easy to refute charges and nearly always side with you on the dispute

**Comment by dillfrescott** [comment]
Might save a lot of headache next time

**Comment by AxDSan** [comment]
Yup, sadly I used my CC, thanks for the heads up

**Comment by AxDSan** [comment]
legit

**Comment by tonydavis629** [comment]
I'm having the same issue. No way to unsubscribe. I have initiated a charge back TWICE. Dev claimed it shouldn't be charging me.
